Blizzard.IO.RabbitMQ/Rpc/RpcRabbitWrapper.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Blizzard.IO.RabbitMQ.Entities.Rpc;
+using EasyNetQ;
+using EasyNetQ.Producer;
+using EesyNetQRpc = EasyNetQ.Producer.Rpc;
+
+namespace Blizzard.IO.RabbitMQ.Rpc
+{
+    public class RpcRabbitWrapper : EesyNetQRpc
```

Comment:
I need you for an explanation on this one too ;)

Reply:
This is letting us control the names of the exchange and the routing key being used for the Rpc communication